root/edit/index.tt

```diff
@@ -60,18 +62,26 @@
             [% END %]
         [% ELSE %]
             <p>
-                [%- l('You must be logged in to vote on edits') -%]
-                [%- request_login() -%]
+                [%- l('You must be logged in to vote on edits.') %]
+                [% request_login() -%]
```

Comment:
OK. However, using `=%]` and `[%=` would still be better because it will squash the whitespace into a single space.

Furthermore, since this is unrelated to the main feature being implemented, this would be perfect for a separate commit (which would also give you room to explain why you are doing this in the commit message).